src/views/registration-js/attendees/fields/birth.php

```diff
@@ -33,47 +33,52 @@
 		<label
 			class="tribe-common-b2--min-medium tribe-tickets-meta-label"
 			for="<?php echo esc_attr( $option_id ); ?>"
-		><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+		>
+			<?php echo wp_kses_post( $field['label'] ); tribe_required_label( $required ); ?>
```

Comment:
* :no_entry_sign: Inline PHP statement must contain a single statement; 2 found
* :no_entry_sign: Each PHP statement must be on a line by itself

<sub>hash: c13f635fb0cf1991f1b2ad7a4408162b</sub>

Reply:
fixed